parachain/runtime/kintsugi/src/lib.rs

```diff
@@ -1114,7 +1114,7 @@ pub type OracleName = oracle::NameOf<Runtime>;
 
 impl oracle::Config for Runtime {
     type RuntimeEvent = RuntimeEvent;
-    type OnExchangeRateChange = vault_registry::PoolManager<Runtime>;
+    type OnExchangeRateChange = (vault_registry::PoolManager<Runtime>, Loans);
```

Comment:
I caught this misconfiguration when running the redeem tests, specifically after liquidation (reimburse) the new exchange rate caused all stake to be invalidated in the capacity pool. This was set correctly on `testnet-kintsugi`: https://github.com/interlay/interbtc/blob/2c3a702eed62a23b25a1c2a1c819a5bdcf67c45d/parachain/runtime/testnet-kintsugi/src/lib.rs#LL1083C12-L1083C12